examples/cmd/snapshotting/example_demo.go

```diff
@@ -301,6 +301,8 @@ func loadSnapshotSSH(ctx context.Context, socketPath, memPath, snapPath, ipToRes
 
 	socketFile := fmt.Sprintf("%s.load", socketPath)
 	cfg := createNewConfig(socketFile, withNetworkInterface(networkInterface))
+	cfg.KernelImagePath = ""
+	cfg.MachineCfg = models.MachineConfiguration{}
```

Comment:
Perhaps consider making these options.

Reply:
I just made a config in place that has those empty, as LoadSnapshot is the only function that would use such an option.

Reply:
Can we remove these two lines then?